_source/_posts/2024-08-15-otp-over-sms.md

```diff
@@ -43,7 +43,7 @@ Soon, [Okta will](https://support.okta.com/help/s/article/bring-your-own-telepho
 
 ## Hooked on telephony
 
-You can send the OTP in the SMS/Voice flow using the [telephony inline hook]([https://help.okta.com/oie/en-us/content/topics/telephony/telephony-inline-hook.htm](https://help.okta.com/oie/en-us/content/topics/telephony/telephony-inline-hook.htm)). Okta uses the code or URL in the hook to send the OTP, though, as you'll see, the hook may not be called every time (and that's a good thing). When your hook fails to send the message or takes too long to update the status, Okta takes over sending the message. However, the number of those messages is heavily rate-limited.
+You can send the OTP in the SMS/Voice flow using the [telephony inline hook](https://help.okta.com/oie/en-us/content/topics/telephony/telephony-inline-hook.htm. Okta uses the code or URL in the hook to send the OTP, though, as you'll see, the hook may not be called every time (and that's a good thing). When your hook fails to send the message or takes too long to update the status, Okta takes over sending the message. However, the number of those messages is heavily rate-limited.
```

Comment:
```suggestion
You can send the OTP in the SMS/Voice flow using the [telephony inline hook](https://help.okta.com/oie/en-us/content/topics/telephony/telephony-inline-hook.htm). Okta uses the code or URL in the hook to send the OTP, though, as you'll see, the hook may not be called every time (and that's a good thing). When your hook fails to send the message or takes too long to update the status, Okta takes over sending the message. However, the number of those messages is heavily rate-limited.
```

Please double check this. I think the syntax is missing the parent and is the link correct?